cdist/conf/type/__consul_template/files/consul-template.sysv

```diff
@@ -34,7 +36,7 @@ start() {
 }
 
 stop() {
-   echo -n "Shutting down $NAME: "
+   printf "Shutting down %s" "$NAME: "
    killproc -p $PID_FILE $CONSUL_TEMPLATE
    retcode=$?
    rm -f /var/lock/subsys/$NAME
```

Comment:
shellcheck isn't complaining about quoting `$NAME` -- but I think it should be. In particular with `rm -f` this could be really nasty with `NAME="foo /etc/passwd"` or similar. OK, NAME is a constant but anyways ...

There are quite some other occurences of `$NAME`, `$PID_FILE`, ... in this file.

This is probably beyond the scope of the current SC work though.